src/Learning/SFA.DAS.Learning.UnitTests/Application/Notifications/WhenGetApprenticeship.cs

```diff
@@ -0,0 +1,91 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Moq;
+using SFA.DAS.Encoding;
+using SFA.DAS.Learning.Application.Notification;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests.Commitments;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses.Commitments;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses.Learning;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.SharedOuterApi.Models;
+using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Learning.UnitTests.Application.Notifications;
+
+[TestFixture]
+internal class WhenGetApprenticeship
```

Comment:
Again in here after the previous rename story around apps inner this is referencing the learning inner api now so should be updated too